src/iam.ts

```diff
@@ -1084,11 +1085,14 @@ export class IAM extends IAMBase {
    * @returns organization with all nested subOrgs
    *
    */
-  getOrgHierarchy({ namespace }: { namespace: string }) {
+  async getOrgHierarchy({ namespace }: { namespace: string }): Promise<IOrganization> {
     if (!this._cacheClient) {
       throw new CacheClientNotProvidedError();
     }
-    return this._cacheClient.getOrgHierarchy({ namespace });
+    const org = await this._cacheClient.getOrgHierarchy({ namespace });
+    [org, ...org.subOrgs || [], ...org.apps || [], ...org.roles || []]
+      .forEach((domain) => domain.isOwned = domain.owner === this.address);
```

Comment:
I find the `isOwned` name to be a little confusing. At first glance, I think it implies that domain is owned at all. Like I think the obvious guess when first seeing the property name is that `isOwned == false` means that the domain isn't owned at all. But it seems what it really means it that isn't owned by the current user. So maybe `isOwnedByCurrentUser` is better, though it is starting to get a little wordy 🤔

Reply:
I agree. This is a more appropriate name. @kim-energyweb , what do you think of this?

Reply:
@JGiter I agree. This is a more concise attribute name.